tests/tween_test.py

```diff
@@ -119,46 +102,6 @@ def test_validation_skips_path_properly():
     assert not should_exclude_path(excluded_paths, '/sample/resources')
 
 
-# TODO: Should probably be migrated to acceptance tests after we make mocking
-# schemas easier there.
-def test_validation_content_type_with_json():
```

Comment:
I'm a little lost - did these already get migrated and so OK to delete?

Reply:
If I understand it correctly, `tween.validate_response` was only used by swagger 1.2 and is deleted in this PR here:
https://github.com/Yelp/pyramid_swagger/pull/248/files#diff-12cf5562318320d89b816f8122b3b384ede6cae269a0a5760d216ce6494465c9L544